implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java

```diff
@@ -82,6 +89,10 @@ public static UncheckedExceptionDataStyle getDefaultUncheckedExceptionDataStyle(
     @Inject
     @ConfigProperty(name = "io.smallrye.health.uncheckedExceptionDataStyle", defaultValue = "ROOT_CAUSE")
     private UncheckedExceptionDataStyle uncheckedExceptionDataStyle = getDefaultUncheckedExceptionDataStyle();
+    
+    @Inject
+    @ConfigProperty(name = "io.smallrye.health.emptyChecksOutcome", defaultValue = "UP")
+    private State emptyChecksOutcome = getDefaultEmptyChecksOutcome();
```

Comment:
Is it necessary to have the method that returns `State.UP` as well as setting the default config value to "UP"?

Reply:
for consistency, I wanted to ensure the default value is set using the producer method in the event that the instance is created outside a micro profile config environment

Reply:
In what situation would you be using SmallRye Health without Config as well?

Reply:
the test cases for one, but really I was just trying to follow along with the existing code base after noticing this: https://github.com/smallrye/smallrye-health/blob/master/implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java#L75-L77